eng/service-templates/ProductConstructionService/provision.bicep

```diff
@@ -176,57 +191,59 @@ resource apiservice 'Microsoft.App/containerApps@2023-04-01-preview' = {
             transport: 'http'
         }
         dapr: { enabled: false }
-        secrets: [
-            {
-                name: 'container-registry-username'
-                value: containerRegistry.listCredentials().username
-            }
-            {
-                name: 'container-registry-password'
-                value: containerRegistry.listCredentials().passwords[0].value
-            }
-        ]
         registries: [
             {
                 server: '${containerRegistryName}.azurecr.io'
-                username: containerRegistry.listCredentials().username
-                passwordSecretRef: 'container-registry-password'
+                identity: identity.id
             }
         ]
       }
-      template: {
-          scale: {
-              minReplicas: 1
-              maxReplicas: 1
-          }
-          serviceBinds: []
-          containers: [ 
-            {
-                image: containerImageName
-                name: 'api'
-                env: env
-                resources: {
-                    cpu: json(containerCpuCoreCount)
-                    memory: containerMemory
-                }
-                probes: [
-                    {
-                        httpGet: {
-                            path: '/health'
-                            port: 8080
-                            scheme: 'HTTP'
-                        }
-                        initialDelaySeconds: 5
-                        periodSeconds: 10
-                        successThreshold: 1
-                        failureThreshold: 3
-                        type: 'Startup'
+        template: {
```

Comment:
Somehow this got a wrong indent by 2 spaces?

Reply:
it was wrong before, I fixed it in this PR

Reply:
![image](https://github.com/dotnet/arcade-services/assets/7013027/63751699-01c5-4419-b953-40cae7254dd1)

Should it not be aligned?

Like here?

![image](https://github.com/dotnet/arcade-services/assets/7013027/2857ab93-a73b-4e02-b916-3926d1fdd6cc)


Just asking - don't know bicep

Reply:
Hm, it doesn't look like that in my vscode, that's weird

Reply:
think I fixed it now